### DISCUSSION TITLE
The scheduler is down or under heavy load

### DISCUSSION THREAD

**Discussion by GeorgeDJKim** [apache/airflow#23483] [General]
### Apache Airflow version

2.2.0

### What happened

All dependencies are met but the task instance is not running. In most cases this just means that the task will probably be scheduled soon unless:
- The scheduler is down or under heavy load

If this task instance does not start soon please contact your Airflow administrator for assistance.

### What you think should happen instead

When the airflow dag is executed, the dag itself succeeds, but the tasks do not work with no_status.

### Operating System

ubuntu 18.04

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Until you provide any details, logs, explamation what happened, and what you have done to diagnose the issue and provide any clues there is no way anyone could help you with it. You need to add more information and do some diagnostics before you reach out with such problems. People here help for free if you provide enough information but, this is not a "support organisation" that woudl solve deployment  problems for you.

**Comment by potiuk** [comment]
Converted it into discussion.
